app/lib/actions/katello/content_view/refresh_rolling_repo.rb

```diff
@@ -0,0 +1,35 @@
+module Actions
+  module Katello
+    module ContentView
+      class RefreshRollingRepo < Actions::EntryAction
+        def plan(repository)
+          action_subject repository
+          sequence do
+            plan_self(repository_id: repository.id)
+            plan_action(Pulp3::Repository::RefreshDistribution, repository, SmartProxy.pulp_primary)
+            plan_action(Repository::IndexContent, id: repository.id, source_repository_id: repository.library_instance.id)
+            plan_action(Actions::Katello::Applicability::Repository::Regenerate, :repo_ids => [repository.id])
+          end
+        end
+
+        def humanized_name
+          _('Refresh RollingCV Repo')
+        end
+
+        def run
+          repository = ::Katello::Repository.find(input[:repository_id])
+          library_instance = repository.library_instance
+          # ensure IndexContent is not skipped!
+          repository.last_contents_changed = DateTime.now if repository.version_href != library_instance.version_href
```

Comment:
```suggestion
          repository.last_contents_changed = DateTime.now if repository.version_href != library_instance.version_href
          repository.clear_smart_proxy_sync_histories
```

This _might_ be the place to fix the smart proxy syncing issue.